test_resolwe_runtime_utils.py

```diff
@@ -15,9 +15,10 @@
 # pylint: disable=missing-docstring
 import json
 from unittest import TestCase
+from unittest.mock import patch
```

Comment:
Yes, I know this. I decided to only support Python 3 since it is available in the main "consumer" of Resolwe Runtime Utilities project, the [Resolwe Bioinformatics Docker container](https://github.com/tjanez/docker-bio-linux8-resolwe).